parser/src/AST/Json.hs

```diff
@@ -335,8 +335,17 @@ instance ToJSON Expr where
                     )
                   ]
 
-          _ ->
-              JSString $ toJSString "TODO: Expr"
+          VarExpr _ ->
+              JSString $ toJSString "TODO: VarExpr"
+
+          Range _ _ _ ->
+              JSString $ toJSString "TODO: Range"
+
+          AccessFunction _ ->
+              JSString $ toJSString "TODO: AccessFunction"
+
+          GLShader _ ->
+              JSString $ toJSString "TODO: GLShader"
```

Comment:
Now's probably a good moment to add a json test (probably just add the json transofmration check to an existing elm fixture test)  for these new things.